README.md

```diff
@@ -45,14 +63,10 @@ go get -u github.com/rogpeppe/godef
 
 You can file any `go-plus` issues [here](https://github.com/joefitzgerald/go-plus/issues/new). You can optionally file an issue in a downstream repository for anything related to a bundled package:
 
-* [`runtime detection`](https://github.com/joefitzgerald/go-config): [create issue](https://github.com/joefitzgerald/go-config/issues/new)
```

Comment:
Clean up: only `go-debug` will be installed, everything else is already part of this package.